src/endpoint/s3/s3_bucket_policy_utils.js

```diff
@@ -136,6 +138,16 @@ async function _is_object_tag_fit(req, predicate, value) {
     dbg.log1('bucket_policy: object tag fit?', value, tag, res);
     return res;
 }
+async function _is_object_version_fit(req, predicate, value) {
+
+    if (!req) {
+        dbg.error('_is_object_version_fit: req is NULL :', req);
+    }
+    const version_id = req.query.versionId;
+    const res = predicate(version_id, value);
+    dbg.error('_is_object_version_fit: Requested version-id, value and result :', version_id, value, res);
```

Comment:
change to dbg.log1. also should probably use the same format as the other condition logs

Reply:
done